sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/operations/_operations.py

```diff
@@ -0,0 +1,1148 @@
+
+        if response.status_code not in [200]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        if cls:
+            return cls(pipeline_response, None, {})
+
+    revoke_permission.metadata = {'url': '/api/hubs/{hub}/permissions/{permission}/connections/{connectionId}'}  # type: ignore
+
+
+    @distributed_trace_async
+    async def check_permission(
+        self,
+        hub: str,
+        permission: str,
+        connection_id: str,
+        *,
+        target_name: Optional[str] = None,
+        api_version: Optional[str] = "2021-08-01-preview",
+        **kwargs: Any
+    ) -> None:
+        """Check if a connection has permission to the specified action.
+
+        Check if a connection has permission to the specified action.
+
+        :param hub: Target hub name, which should start with alphabetic characters and only contain
+         alpha-numeric characters or underscore.
+        :type hub: str
+        :param permission: The permission: current supported actions are joinLeaveGroup and
+         sendToGroup. Possible values are: "sendToGroup" or "joinLeaveGroup".
+        :type permission: str
+        :param connection_id: Target connection Id.
+        :type connection_id: str
+        :keyword target_name: The meaning of the target depends on the specific permission. For
+         joinLeaveGroup and sendToGroup, targetName is a required parameter standing for the group name.
+        :paramtype target_name: str
+        :keyword api_version: Api Version.
+        :paramtype api_version: str
+        :return: None
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        cls = kwargs.pop('cls', None)  # type: ClsType[None]
+        error_map = {
+            401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError
+        }
+        error_map.update(kwargs.pop('error_map', {}))
+
+        
+        request = build_web_pub_sub_check_permission_request(
+            hub=hub,
+            permission=permission,
+            connection_id=connection_id,
+            target_name=target_name,
+            api_version=api_version,
+            template_url=self.check_permission.metadata['url'],
+        )
+        request.url = self._client.format_url(request.url)
+
+        pipeline_response = await self._client.send_request(request, stream=False, _return_pipeline_response=True, **kwargs)
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200, 404]:
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        if cls:
+            return cls(pipeline_response, None, {})
+
+    check_permission.metadata = {'url': '/api/hubs/{hub}/permissions/{permission}/connections/{connectionId}'}  # type: ignore
+
```

Comment:
the whole `core` folder should be deleted, we don't need to vendor azure-core in the sdk anymore

Reply:
it seems the Github display bug. `core` folder has been deleted.